internal/services/appconfiguration/app_configuration_resource.go

```diff
@@ -665,6 +786,41 @@ func expandAppConfigurationEncryption(input []interface{}) *configurationstores.
 	return result
 }
 
+func expandAppConfigurationReplicas(input []interface{}, configurationStoreName, configurationStoreLocation string) ([]replicas.Replica, error) {
+	result := make([]replicas.Replica, 0)
+
+	locationSet := map[string]interface{}{
+		configurationStoreLocation: nil,
+	}
```

Comment:
What is the purpose of this?

Reply:
This is to using Map in golang to check each replica's location is unique, set `nil` is because the value of Map k-v doesn't matter here. Have changed value to `name of replica` now, for better error message if there are duplicated locations.